### PR TITLE
Feat/insert new mlp

### PR DESCRIPTION
**Pull request by DonatasTamosauskas**
Working functionality of inserting a new MLP record. If no date is inserted, current date is sent to the server.

### CODE REVIEW

**Comment by DonatasTamosauskas** [comment]
HUNDREATH PULL REQUEST BOIS

File: HucaresReact/src/pages/MLP/MLPTable.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { chunkArray } from '../../utils/Array';
 import PaginationContainer from '../../components/Pagination/Pagination';
 import { MLPDeleteModal } from '../../components/Modal/DataHelpers/MLPHelpers/MLPDeleteModal';
 import { InfoEditingModal } from '../../components/Modal/InfoEditingModal';
+import Button from "../../../node_modules/semantic-ui-react/dist/commonjs/elements/Button/Button";
```
Comment (SSMellow) [inline HucaresReact/src/pages/MLP/MLPTable.jsx]: What the hell is this and where you are using it?

File: HucaresReact/src/components/Modal/InfoEditingModal.jsx [hunk 2]
```diff
@@ -7,21 +8,67 @@ export class InfoEditingModal extends React.Component {
...
+          .post(`${process.env.HUCARES_API_BASE_URL}/api/mlp/insert`,
+              {
+                  headers: { 'Access-Control-Allow-Origin': '*' },
+                  plateNumber: this.state.forms[0].value,
+                  searchStartDatetime: this.convertDate(this.state.forms[1].value)
+              })
+          .then((response) =>
+            {
+              this.forceUpdate();
+              if(this.state.callback !== undefined) this.state.callback();
```
Comment (SSMellow) [inline HucaresReact/src/components/Modal/InfoEditingModal.jsx]: I think `if(this.state.callback) this.state.callback();` would be enough, but not sure

File: HucaresReact/src/pages/MLP/MLPTable.jsx (hunk 1 above)
Comment (DonatasTamosauskas) [reply HucaresReact/src/pages/MLP/MLPTable.jsx]: Legacy code, will remove

**Comment by DonatasTamosauskas** [comment]
Issues fixed

**Pull request merged**